src/datetimepicker.android.js

```diff
@@ -109,18 +123,16 @@ export default function RNDateTimePicker(props: AndroidNativeProps) {
 
           switch (action) {
             case DATE_SET_ACTION:
-              event.nativeEvent.timestamp = date.setFullYear(year, month, day);
+              date.setFullYear(year, month, day);
+              date.setHours(0, 0);
```

Comment:
Why does the time get reset to 0 when the date changes?

Reply:
Hello, it's possible this is a bug. If you open a PR that explains and fixes the issue, I'll merge it. Thanks!

Reply:
Why would we ever want to keep the time? We are on a date picker, and I assumed we would clear out the time?

Reply:
@billnbell In the scenario that you have a separate date field and time field, you wouldn't want the time field to reset every time you change the date field.